README.md

```diff
@@ -1,29 +1,59 @@
+## Development
 
-The spec is written using [Bikeshed](https://tabatkins.github.io/bikeshed). Please refer to the [Bikeshed Documentation](https://tabatkins.github.io/bikeshed/) for the Bikeshed syntax.
+This section describes the process for building this specification on your local computer for development and testing purposes. You may want to do this before you submit a [Pull Request](#pull-requests) to catch possible build issues early.
 
-For testing purposes, you can generate the spec locally:
+The first step is to clone the repo:
 
-If you have bikeshed [installed locally](https://tabatkins.github.io/bikeshed/#installing), you can generate the specification locally with:
+```
+prompt> gh repo clone webmachinelearning/webnn
+```
+
+Next, install the makefile dependencies:
 
 ```
-prompt> make
+prompt> cd tools
+prompt> npm install
+prompt> cd ..
 ```
 
-This simply runs bikeshed on the `index.bs` file.
+Now you can choose between the two options:
+
+- [Install Bikeshed locally](https://speced.github.io/bikeshed/#install-final) (works offline, faster build) and run:
+
+```
+prompt> make
+```
 
-Otherwise, you can use the [bikeshed Web API](https://tabatkins.github.io/bikeshed/#remote):
+- Or use the [Bikeshed HTTP API](https://speced.github.io/bikeshed/#remote) (requires internet, slower build):
 
 ```
 prompt> make online=1
 ```
+
+In both the cases, the makefile runs [tools](tools) to check the source formatting and catch common errors, and then uses Bikeshed to convert the `index.bs` source file to an `index.html` output file.
+
+If the build is clean you can open the `index.html` file in your browser to view it in all its glory! If you're happy with what you see, congratulations! You can proceed to submit a [Pull Request](#pull-requests).
+
+## Pull Requests
+
+First, please read the [contributing guidelines](CONTRIBUTING.md).
+
+The spec is written using [Bikeshed](https://speced.github.io/bikeshed/) spec-generating tool. Please refer to the [Bikeshed Documentation](https://speced.github.io/bikeshed/) for the Bikeshed syntax and [coding conventions](docs/SpecCodingConventions.md) specific to this specification.
+
+To propose a change to the specification, you are only required to update the `index.bs` source file authored in Bikeshed syntax.
+
+Before you submit a PR, it is recommended to test your changes locally (see [Development](#development)). If your local build is clean, you can proceed to submit the PR. If you want to submit a WIP PR that still contains errors or is incomplete, please [convert the PR to a draft](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/changing-the-stage-of-a-pull-request#converting-a-pull-request-to-a-draft). As a convention, please [link the PR to an existing issue](https://graphite.dev/guides/how-to-link-pull-requests-to-github-issues), if any.
```

Comment:
> As a convention, please [link the PR to an existing issue](https://graphite.dev/guides/how-to-link-pull-requests-to-github-issues), if any.

According to https://github.com/webmachinelearning/webnn/blob/main/CONTRIBUTING.md#type-of-change, PRs of type Bug fix and New Content are required to link to an GitHub issue.

Reply:
I added a direct link to that section to clarify expectations, https://github.com/webmachinelearning/webnn/pull/852/commits/d64dfc510333e5d954e13259c8659b1d68da1e55